addr/bitcoin.go

```diff
@@ -0,0 +1,19 @@
+package addr
+
+import (
+	"github.com/bitlum/btcd/chaincfg"
+	"github.com/bitlum/btcutil"
+	"github.com/go-errors/errors"
+)
+
+// validateBitcoin validates bitcoin address according net. Return error if address is invalid.
```

Comment:
`Return error if address is invalid`, we have schema for this.

Reply:
You mean this commend is superflous?

Reply:
yep